01-Login/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:4.2-jdk8
+
+WORKDIR /tmp
+ADD . /tmp
+
+RUN gradle build
+
+CMD java -jar build/libs/auth0-spring-mvc.war
```

Comment:
We [are telling users](https://github.com/auth0-samples/auth0-spring-mvc-sample/tree/master/01-Login#running-the-sample) to run the sample using `bootRun` as that's the task that spring boot adds to the project. What's the benefit on doing it this other way?